packages/calling/src/Metrics/index.ts

```diff
@@ -321,6 +321,43 @@ class MetricManager implements IMetricManager {
       this.webex.internal.metrics.submitClientMetrics(name, data);
     }
   }
+
+  public submitBNRMetric(
+    name: METRIC_EVENT,
+    metricAction: string,
+    type: METRIC_TYPE,
+    callId: CallId,
+    correlationId: CorrelationId
+  ) {
+    let data;
+
+    if (METRIC_EVENT.MEDIA) {
```

Comment:
Is this if condition necessary?

Reply:
I believe this was added to differentiate the error flow from the success one